apps/hub-app/src/components/Profile.tsx

```diff
@@ -54,7 +54,7 @@ const Profile = () => {
           image: profile.image || '',
           description: profile.description || '',
           emoji: profile.emoji || '',
-          background: profile.background || '',
+          background: profile.image || '',
```

Comment:
FYI - I've removed this field from the SDK. Looks like it was removed from the designs, right?

Reply:
Ah no worries, going forward this pr should raise those issues as the changes are made.